src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static Storage instance;
+    private final Map<String, FruitTransaction> fruitInventory;
```

Comment:
```suggestion
    private final Map<String, Integer> fruitInventory;
```
We are keeping Fruits on the storage - not FruitTransactions